juha/programmers/3/두개-뽑아서-더하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(numbers) {
+  total = numbers.length;
+  let answer = new Set();
```

Comment:
new 생성자로 set 타입을 사용할 때 const 로 선언해도 데이터 가공에 무방합니다.
```suggestion
  const answer = new Set();
```